pkg/virtctl/pause/pause.go

```diff
@@ -59,15 +61,18 @@ First argument is the resource type, possible types are (case insensitive, both
 Second argument is the name of the resource.`,
 		Args:    templates.ExactArgs(COMMAND_PAUSE, 2),
 		Example: usage(COMMAND_PAUSE),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			c := VirtCommand{
-				command:      COMMAND_PAUSE,
-				clientConfig: clientConfig,
-			}
-			return c.Run(args)
-		},
 	}
+
+	var dryRun bool
 	cmd.Flags().BoolVar(&dryRun, "dry-run", false, "--dry-run=false: Flag used to set whether to perform a dry run or not. If true the command will be executed without performing any changes.")
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
```

Comment:
Can you do it e.g. like here https://github.com/kubevirt/kubevirt/blob/82779585515c4df2945dea8980ee86c7749f6d93/pkg/virtctl/create/vm/vm.go#L274?

Use the structs members for the Flags?

Reply:
Done